light_switch.js

```diff
@@ -1 +1,17 @@
 // code here
+// number
+// var num = 3;
+// boolean
+// var lightOn = false;
+// charector
+// var car = "E";
+// string
+// var string = "Mohib is hansom";
+
+var lightOn = false;
+if(lightOn === false) {
+  document.write('the light on' + '</br >');
+}
```

Comment:
There is a convention on how to write if-else function:
```
// good
if (test) {
  console.log("I won");
} else {
  console.log("I lost");
}
```

In airbnb style guide is a good chapter about white spaces, which covers this too.

https://github.com/airbnb/javascript#whitespace